app/controllers/spree/admin/base_controller_decorator.rb

```diff
@@ -34,7 +34,7 @@ def unauthorized
       redirect_to '/unauthorized'
     else
       store_location
-      redirect_to root_path(anchor: "login?after_login=#{ request.env['PATH_INFO'] }")
+      redirect_to root_path(anchor: "login?after_login=#{request.env['PATH_INFO']}")
     end
   end
 
```

Comment:
This reminds me - I had problems with getting signed out of OFN when going through the Stripe Connect process to connect an account. When I was redirected back to OFN I got prompted to sign in, but was not then redirected to the admin page (and the account was not connected on the OFN side as the request was not successful). I only experienced it locally and I'm not sure what the reason for getting signed out was. I remember playing with using some of the other request params here instead (e.g. #fullpath), but was worried I was going to break everything...

https://github.com/stveep/openfoodnetwork/commit/9f3888c68e627a8fa3fbfc7ed13bebf0775e04e5